src/core/99_variables.tf

```diff
@@ -443,6 +455,13 @@ variable "eucovidcert_alerts_enabled" {
 }
 
 # app backend
+
+variable "app_backend_names" {
+  description = "App backend instance names"
+  type        = list(string)
+  default     = ["appbackendl1", "appbackendl2", "appbackendli"]
```

Comment:
please can you put this values into the env folder

Reply:
Sure, done in [fix after review](https://github.com/pagopa/io-infra/pull/296/commits/d04ab9cea130bcf3dcbebbf4d7a819d4ab95b1d1)